checks/lint-config.json

```diff
@@ -1,7 +1,4 @@
 {
     "MD004": false,
-    "MD013": {
-        "code_blocks": false,
-        "tables": false
-    }
+    "MD013": false
```

Comment:
As it was largely ignored I turned off this rule entirely. Happy to revert this ( and fix the line breaks everywhere 😅 )